lib/pages/editor-page.js

```diff
@@ -104,7 +104,7 @@ export default class EditorPage extends BaseContainer {
 
 	dismissMediaModal() {
 		const driver = this.driver;
-		const cancelSelector = webdriver.By.xpath( "//div[contains(@class, 'editor-media-modal')]//span[contains(@class, 'dialog__button-label') and contains(text(), 'Cancel' )]" );
+		const cancelSelector = webdriver.By.css( 'div.dialog__action-buttons > button:nth-child(1) > span' );
```

Comment:
Thanks for the suggestions!

I ended up going with the data-e2e attribute, which is now in a PR over on the calypso project,
 [here](https://github.com/Automattic/wp-calypso/pull/14075).

I'll be pushes changes shortly that use this new attribute.